meals/templates/include/field.html

```diff
@@ -0,0 +1,4 @@
+{% if value %}
+    <dt>{{ desc }} {% if showlength %} ({{ value | length}}) {% endif %}</dt>
```

Comment:
You should be able to use the already existing template for this instead of duplicating it.

Reply:
Hmm, if it's in ``app`` then I don't need to have this, right?

Reply:
True, removed.